semestre-service/src/main/java/ma/enset/semestreservice/controller/SemestreController.java

```diff
@@ -61,6 +61,14 @@ public ResponseEntity<SemestreResponse> get(@PathVariable("codeSemestre") String
                 .ok()
                 .body(foundSemestreResponse);
     }
+    @PostMapping("/GetBulkByCode")
```

Comment:
- We use the `GET` method to retrieve data
- We use nouns instead of verbs for the endpoints. In this case it would be `/bulk` instead of '/GetBulkByCode'

Reply:
mabantch lk tmak wa7d  @bodyresponse , kiba4i dir liha f @get

Reply:
kayna tmak wa7d @requestbody , ki4adoz f @getmapping ?